yarn.lock

```diff
@@ -7446,6 +7447,16 @@ import-in-the-middle@1.7.1:
     cjs-module-lexer "^1.2.2"
     module-details-from-path "^1.0.3"
 
+import-in-the-middle@^1.10.0:
```

Comment:
try running `npx yarn-deduplicate && yarn install` and see if this one goes away

Reply:
Just ran